packages/components/src/types/page.ts

```diff
@@ -33,16 +33,21 @@ const BaseCollectionItemPageSchema = Type.Object({
   ),
 })
 
-const BaseRefPageSchema = Type.Composite([
-  BaseCollectionItemPageSchema,
-  Type.Object({
-    ref: Type.String({
-      title: "URL to the actual item",
-      description:
-        "The link that users will open immediately when they click on the item in the parent collection page",
-    }),
+const BaseRefPageSchema = Type.Object({
+  ref: Type.String({
+    title: "Link",
+    description: "Choose a page or file to link to this Collection item",
+    format: "ref",
   }),
-])
+  summary: Type.Optional(
+    Type.String({
+      title: "Summary",
+      description: "This will appear on the collection",
+      format: "textarea",
+      maxLength: 120,
+    }),
+  ),
```

Comment:
issue: need to check this but I think this might not work because we generate the sitemap using either `subtitle` or `description`.

https://github.com/opengovsg/isomer/blob/bc3e2f2ee00518c686d2da488de5ed7ac2f05149/tooling/build/scripts/generate-sitemap.js#L56-L63

This should be called `description` instead.

Reply:
fixed